packages/kitten-scientists/source/ui/UserInterface.ts

```diff
@@ -117,7 +117,15 @@ export class UserInterface extends UiComponent {
     $("#clearLog").prepend(this.showActivity);
 
     // Add Kitten Scientists above the game log.
-    const right = $("#rightColumn");
+    const id = localStorage["ks.Column"] as string | undefined;
```

Comment:
Read the state from the settings. Then pick either #rightColumn or #ksColumn.

Reply:
![image](https://github.com/kitten-science/kitten-scientists/assets/38437979/42e5b1c1-7d90-439c-b904-1a724c761d0b)
When entering the web page, UI drawing is executed before loading the settings, and the correct settings cannot be executed.

Reply:
Maybe redraw the UI after loading the settings? let me try

Reply:
Maybe this is similar to the language setting. Check out if the same approach should be used here:

https://github.com/kitten-science/kitten-scientists/blob/4d7f05981a93783f6b3b842e515012b49ba13b54/packages/kitten-scientists/source/Engine.ts#L149-L151